spouts/rss/feed.php

```diff
@@ -212,7 +212,7 @@ public function getId() {
      */
     public function getTitle() {
         if ($this->items !== false && $this->valid()) {
-            return @current($this->items)->get_title();
+            return htmlspecialchars_decode(@current($this->items)->get_title());
```

Comment:
Are you sure? From a quick test, the behavior for this feed is the same as before, which was my intention. A few years ago, some people were expecting HTML in the title. No sure what to do here.